File: UI/EAgroUIApp/src/app/Models/verifiedcollection.ts
export class Verifiedcollection {
    constructor(public id:number,public collectionCenterName:string, 
        public collectionCenterId:number,public corporateId:number,
        public cropName:string,public imageUrl:string,public containerType:string,
        public quantity:number,public weight:number,public collectionDate:string,
        public grade:string,public verifiedWeight:number,public inspectionDate:string,
        public paymentStatus: string,
        public inspectorId:number){}
}

File: API/VerifiedGoodsCollections/Models/VerifiedCollection.cs
namespace VerifiedGoodsCollections.Models;
public class VerifiedCollection{
    public int Id{get;set;}
    public int CollectionId{get;set;}
    public string? Grade{get;set;}
    public double Weight{get;set;}
    public int InspectorId{get;set;}
    public DateTime InspectionDate{get;set;}

    public VerifiedCollection(){
        InspectionDate=DateTime.Now;
    }
}